src/lucky_migrator/create_table_statement.cr

```diff
@@ -113,6 +114,28 @@ class LuckyMigrator::CreateTableStatement
     raise "index on #{@table_name}.#{column} already exists"
   end
 
+  # Adds a references column and index given a model class and references option.
+  macro belongs_to(model_class, references = nil)
+    {% optional = model_class.is_a?(Generic) %}
+
+    {% if optional %}
+      {% class_underscore = model_class.type_vars.first.stringify.underscore %}
+    {% else %}
+      {% class_underscore = model_class.names.first.stringify.underscore %}
+    {% end %}
+
+    {% foreign_key_name = class_underscore + "_id" %}
+    %table_name = {{ references }} ? {{ references }} : pluralize({{ class_underscore }})
+
+    add_column {{ foreign_key_name }}, Int64, {{ optional }}, %table_name
+    add_index {{ foreign_key_name }}
+  end
+
+  def pluralize(word : String)
+    return "#{word.downcase.rchop}ies" if word.ends_with?("y")
+    return "#{word.downcase}s"
```

Comment:
Oh nice. I like hat you handled the `"y"` option here. One request, could you do an `if` here to keep things consistent with the rest of the lib?

```crystal
if word.ends_with?("y")
  "#{word.downcase.rchop}ies"
else
  "#{word.downcase}s"
end
  
```

Reply:
Oh and I don't think you need to downcase here. I *believe* underscore does that. But if not, I'd do the downcase in the `belongs_to` macro instead of in `pluralize`.

Reply:
you're right, I think underscore takes care of it